ssh-webservice.php

```diff
@@ -2,8 +2,14 @@
 header('Content-Type: text/plain');
 
 $user = 'maxux';
-if($_SERVER['REQUEST_URI'] != '/')
-	$user = substr($_SERVER['REQUEST_URI'], 1);
+$expr = 'END';
+if($_SERVER['REQUEST_URI'] != '/') {
+    $paths = explode("/", $_SERVER['REQUEST_URI']);
+    $user = $paths[1];
+    if (count($paths) > 2) {
+        $expr = 'NR==' . $paths[2];
```

Comment:
I would not allows this kind of unsafe call. Maybe some int() parsing to ensure integer.
This could provide some command injection.